spec/lib/data_pull_spec.rb

```diff
@@ -170,14 +170,14 @@
       subject(:result) { subtask.run(args:, config:) }
 
       it 'looks up the UUIDs for the given email addresses', aggregate_failures: true do
-        expect(result.table).to eq(
-          [
-            ['email', 'uuid'],
-            *users.map { |u| [u.email_addresses.first.email, u.uuid] },
-            ['missing@example.com', '[NOT FOUND]'],
-          ],
-        )
-
+        expected_table = [
+          ['email', 'uuid'],
+          *users.map { |u| [u.email_addresses.first.email, u.uuid] },
+          ['missing@example.com', '[NOT FOUND]'],
+        ]
+
+        expect(result.table).to eq(expected_table)
```

Comment:
I'm probably just being dense, but I don't entirely follow how these changes would have caught the previous issue. Shouldn't an equality check have failed?

Reply:
You're right, I my added check does not address the issue.

Reply:
Added a hopefully clearer and more correct check in 3500b6c037